mantidproject/mantid
# Indirect Bayes Quasi - Error when using resnorm file

Fixes #15062

**Identified in unscripted testing using Mantid nightly build 20th Jan**

Quasi interface should now allow for _ResNorm files to be used properly. The tests have also been updated to reflect this.

**Files are available from `\ExternalData\Testing\Data\UnitTest`**
# To Test
- Run the following script (you will have to have `\ExternalData\Testing\Data\UnitTest`in your directory path):

```
irs26176_graphite002_red = Load(Filename="irs26176_graphite002_red")
irs26173_graphite002_res  = Load(Filename="irs26173_graphite002_res")
irs26173_graphite002_ResNorm = Load(Filename="irs26173_graphite002_ResNorm")

fit_group, result, prob = BayesQuasi ( Program='QL',
                                                   SampleWorkspace=irs26176_graphite002_red,
                                                   ResolutionWorkspace=irs26173_graphite002_res,
                                                   ResNormWorkspace=irs26173_graphite002_ResNorm,
                                                   MinRange=-0.547607,
                                                   MaxRange=0.543216,
                                                   SampleBins=1,
                                                   ResolutionBins=1,
                                                   Elastic=False,
                                                   Background='Sloping',
                                                   FixedWidth=False,
                                                   UseResNorm=True,
                                                   WidthFile='',
                                                   Loop=True,
                                                   Save=False,
                                                   Plot='None')
```
- Ensure the following:
  - There are 6 workspaces ending in:
    - `_res` (Matrix)
    - `_ResNorm` (Group)
    - `_QLr_Prob` (Matrix)
    - `_QLr_Result` (Matrix)
    - `_QLr_Workspaces` (Group)
    - `_red` (Matrix)
  - QLr workspace has 10 workspaces names ending in `0-9`
  - The new updated unit tests pass
